lib/modules/styleTweaks.js

```diff
@@ -406,81 +392,6 @@ modules['styleTweaks'] = {
-			if (thisExpando.classList.contains('twitterLoaded')) {
-				thisExpando.style.display = 'block';
-				return;
-			}
-			var twitterLink = e.target.previousSibling.querySelector('.title');
-			if (twitterLink) twitterLink = twitterLink.getAttribute('href').replace('/#!', '');
-			var match = twitterLink.match(/twitter.com\/[^\/]+\/(?:status|statuses)\/([\d]+)/i);
-			if (match !== null) {
-				// var jsonURL = 'http://api.twitter.com/1/statuses/show/'+match[1]+'.json';
-				var jsonURL = 'https://api.twitter.com/1/statuses/oembed.json?id=' + match[1],
-					thisJSON = {
-						requestType: 'loadTweet',
-						url: jsonURL
-					};
-				if (BrowserDetect.isChrome()) {
-					// we've got chrome, so we need to hit up the background page to do cross domain XHR
-
-					// first, we need to see if we have permissions for the twitter API...
-					var permissionsJSON = {
-						requestType: 'permissions',
-						callbackID: permissionQueue.count,
-						data: {
-							origins: ['https://api.twitter.com/*']
-						}
-					};
-					// save a function call that'll run the expando if our permissions request
-					// comes back with a result of true
-					permissionQueue.onloads[permissionQueue.count] = function(hasPermission) {
-						if (hasPermission) {
-							chrome.runtime.sendMessage(thisJSON, function(response) {
-								// send message to background.html
-								var tweet = response;
-								$(thisExpando).html(tweet.html);
-								thisExpando.style.display = 'block';
-								thisExpando.classList.add('twitterLoaded');
-							});
-						} else {
-							// close the expando since we don't have permission.
-							$(e.target).removeClass('expanded').addClass('collapsed collapsedExpando');
-						}
-					};
-					permissionQueue.count++;
-
-					// we do a noop in the callback here because we can't actually get a
-					// response - there's multiple async calls going on...
-					chrome.runtime.sendMessage(permissionsJSON, function(response) {});
-
-				} else if (BrowserDetect.isSafari()) {
-					// we've got safari, so we need to hit up the background page to do cross domain XHR
-					modules['styleTweaks'].tweetExpando = thisExpando;
-					safari.self.tab.dispatchMessage(thisJSON.requestType, thisJSON);
-				} else if (BrowserDetect.isOpera()) {
-					// we've got opera, so we need to hit up the background page to do cross domain XHR
-					modules['styleTweaks'].tweetExpando = thisExpando;
-					opera.extension.postMessage(JSON.stringify(thisJSON));
-				} else if (BrowserDetect.isFirefox()) {
-					// we've got a jetpack extension, hit up the background page...
-					// we have to omit the script tag and all of the nice formatting it brings us in Firefox
-					// because AMO does not permit externally hosted script tags being pulled in from
-					// oEmbed like this...
-					jsonURL += '&omit_script=true';
-					modules['styleTweaks'].tweetExpando = thisExpando;
-					self.postMessage(thisJSON);
```

Comment:
> AMO does not permit externally hosted script tags being pulled in from oEmbed like this
The thing is, we're adding `&omit_script=true` to the original `jsonURL`, which doesn't affect the URL used by our request (in thisJSON, which has already been created [here](https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/2133/files#diff-6360b0be2d4a48c79cdcd07e6d8c9820L423)).

So we need to decide if we want to correctly omit the script (current behavior; loses twitter formatting in FF) or keep the same behavior and load it anyways.